src/components/tooltip/tooltip-styles.jsx

```diff
@@ -69,70 +150,16 @@ export default theme => {
         zIndex: zIndexTooltip,
       },
 
-      '&$above': {
-        left: '50%',
-        bottom: '100%',
-        transform: 'translateX(-50%)',
-        marginBottom: 8,
-        '&:before': {
-          bottom: -8,
-          borderTop: `8px solid ${palette.text.default}`,
-          borderRight: '8px solid transparent',
-          borderLeft: '8px solid transparent',
-        },
-      },
-
-      '&$below': {
-        left: '50%',
-        transform: 'translateX(-50%)',
-        marginTop: 8,
-        '&:before': {
-          top: -8,
-          borderLeft: '8px solid transparent',
-          borderRight: '8px solid transparent',
-          borderBottom: `8px solid ${palette.text.default}`,
-        },
-      },
-
-      '&$left': {
-        right: 8,
-        top: '50%',
-        marginRight: '100%',
-        transform: 'translate(0, -50%)',
-        '&:before': {
-          left: 'inherit',
-          right: -8,
-          marginTop: -6,
-          top: '50%',
-          content: '""',
-          borderTop: '6px solid transparent',
-          borderBottom: '6px solid transparent',
-          borderLeft: `6px solid ${palette.text.default}`,
-        },
-      },
-
-      '&$right': {
-        left: 8,
-        top: '50%',
-        marginLeft: '100%',
-        transform: 'translate(0, -50%)',
-        '&:before': {
-          right: 'inherit',
-          left: -2,
-          marginTop: -6,
-          top: '50%',
-          content: '""',
-          borderTop: '6px solid transparent',
-          borderBottom: '6px solid transparent',
-          borderRight: `6px solid ${palette.text.default}`,
-        },
-      },
+      ...popupPlacementStyles(palette),
+      ...wrapInMobileMediaQuery(mixins, popupPlacementStyles(palette, 'Mobile')),
+      ...wrapInDesktopMediaQuery(mixins, popupPlacementStyles(palette, 'Desktop')),
     },
     popupFixed: {
       whiteSpace: 'inherit',
     },
 
     popupContent: {
+      position: 'static',
```

Comment:
it is the default one, isn't it?

Reply:
You're right, I moved out an inline style without thinking about it.